Makefile

```diff
@@ -56,7 +56,13 @@ all: install build
 install:
 	git lfs install
 	go mod download
+
+ifneq ($(OS),Windows_NT)
+	# On GH windows runners this fails at the moment: https://github.com/actions/runner-images/issues/10009
+	# since we don't run the linter on windows in the CI, we can skip this step for now
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s $(GOLANGCI_LINT_VERSION)
+endif
+
```

Comment:
@fasmat It's not only used in CI but also by devs on Windows (IDK if there is anybody developing on Windows at present). Now `make lint` will fail with "command not found" error for them...